doc/user_manual/PostProcessors/PhysicsGuidedCoverageMapping.tex

```diff
@@ -0,0 +1,32 @@
+\label{PhysicsGuidedCoverageMapping}
```

Comment:
move to Validation.tex

Reply:
Content moved to Validation.tex and the separate .tex file removed.